torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -876,6 +886,241 @@ def get_b_layout(self):
         return LayoutType.VNNI2
 
 
+@register_micro_gemm(
+    *generate_gemm_config(
+        VecAVX512,
+        [(4, 64, 32), (4, 64, 64), (4, 64, 128)],
+        input_dtype=torch.bfloat16,
+        input2_dtype=torch.uint8,
+        output_dtype=torch.float,
+        compute_dtype=torch.float,
+    ),
+)
+class CppMicroGemmWoQInt4Vec(CppMicroGemmFP32Vec):
```

Comment:
@jgong5 had asked if we could reuse `CppMicroGemmFP32Vec` & modify it to support int4 WoQ GEMM. Would that be possible? Thanks

Reply:
WOQ INT4 microkernel has many specific codes. I think it would be easier to maintain by separating them. Thanks.

Reply:
Thanks! I agree that combining both would make the code messy.